bin/typescript-fetch-petstore-interfaces.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
please add a .cmd file for windows users

Reply:
@macjohnny is this resolved? Are there any more updates I should do for this PR?

Reply:
@tsiq-jeremy the .cmd file was added, so this particular task is resolved. 
so far, there are no other change requests, so maybe someone else will also approve this PR and @wing328 will eventually merge it.